docker/install-kokkos.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+cd /tmp
+
+git clone -b 4.3.00 https://github.com/kokkos/kokkos
+cd kokkos/
+mkdir build
+cd build/
+cmake ..  -D Kokkos_ENABLE_CUDA=OFF -DKokkos_ENABLE_OPENMP=ON
+make -j $(nproc) install
+cd
```

Comment:
Do we need that `cd`?

Reply:
The goal here is to change directory to a safe folder like $HOME before we erase the folder we are currently in. I'm not entirely sure what happens to commands that open files for writing like `ldconfig` when the current working directory is unlinked.